From 6c4c94b545ca2bd757df8a2e96516c96b5958e39 Mon Sep 17 00:00:00 2001
From: Andrey Konovalov <andrey.konovalov@linaro.org>
Date: Fri, 17 May 2013 23:55:18 +0400
Subject: [PATCH] configs: Enable NFS in linaro-base.conf

As per launchpad bug 1167446, adding NFS is requested for arndale.
vexpress[64], panda and snowball have NFS enabled in their board config
fragments already. This is natural just to move it to linaro-base.conf.

Signed-off-by: Andrey Konovalov <andrey.konovalov@linaro.org>
---
 linaro/configs/linaro-base.conf | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/linaro/configs/linaro-base.conf b/linaro/configs/linaro-base.conf
index 5c748a7..97576df7 100644
--- a/linaro/configs/linaro-base.conf
+++ b/linaro/configs/linaro-base.conf
@@ -69,6 +69,12 @@ CONFIG_JFFS2_COMPRESSION_OPTIONS=y
 CONFIG_JFFS2_LZO=y
 CONFIG_JFFS2_RUBIN=y
 CONFIG_CRAMFS=y
+CONFIG_NETWORK_FILESYSTEMS=y
+CONFIG_NFS_FS=y
+CONFIG_NFS_V3=y
+CONFIG_NFS_V3_ACL=y
+CONFIG_NFS_V4=y
+CONFIG_ROOT_NFS=y
 CONFIG_NLS_CODEPAGE_437=y
 CONFIG_NLS_ISO8859_1=y
 CONFIG_PRINTK_TIME=y
-- 
1.8.0

